src/routes/Login.jsx

```diff
@@ -1,17 +1,81 @@
 import HeadingA from "../components/HeadingA";
-import TextBox from "../components/TextBox";
+import EmailBox from "../components/EmailBox";
+import PasswordBox from "../components/PasswordBox";
 import AnimatedButton from "../components/AnimatedButton";
+import api from "../api";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { REFRESH_TOKEN } from "../constants";
+
+const LOGIN_URL = "/auth/login";
 
 export default function Login() {
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await api.post(
+        LOGIN_URL,
+        JSON.stringify({
+          email: email,
+          password: password,
+        }),
+        {
+          headers: {
+            "Content-Type": "application/json",
+          },
+          withCredentials: true,
+        }
+      );
+      if (response.status === 200) {
+        // saving refresh token in local storage
+        localStorage.setItem(REFRESH_TOKEN, response.data.refresh_token);
```

Comment:
Replace refresh_token with refreshToken (You could also see the response in api-docs for help)